.docker/qgis3-qt6-build-deps.dockerfile

```diff
@@ -72,7 +72,7 @@ RUN dnf -y --refresh install \
     qtkeychain-qt6-devel \
     qwt-qt6-devel \
     qscintilla-qt6-devel \
-    sip6 \
+    sip6-6.8.6-2.fc41.noarch \
```

Comment:
After testing on my laptop, it works without specifying the version

Reply:
@benoitdm-oslandia THX for sharing this insight :+1: